From 2e9547d93456722db9e4336ea0b2ab8dfc3f2ade Mon Sep 17 00:00:00 2001
From: Lorenz Brun <lorenz@brun.one>
Date: Mon, 21 Mar 2022 16:11:14 +0100
Subject: [PATCH 2/2] Fix googleapis import paths

Generated using this sed expression:
s#@org_golang_google_genproto//googleapis/#@go_googleapis//google/#
---
 cel/BUILD.bazel               |  4 ++--
 checker/BUILD.bazel           |  2 +-
 checker/decls/BUILD.bazel     |  2 +-
 common/BUILD.bazel            |  2 +-
 common/containers/BUILD.bazel |  4 ++--
 common/debug/BUILD.bazel      |  2 +-
 common/types/BUILD.bazel      |  6 +++---
 common/types/pb/BUILD.bazel   |  2 +-
 common/types/ref/BUILD.bazel  |  2 +-
 ext/BUILD.bazel               |  2 +-
 interpreter/BUILD.bazel       |  4 ++--
 parser/BUILD.bazel            |  2 +-
 server/BUILD.bazel            | 12 ++++++------
 test/BUILD.bazel              |  2 +-
 14 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/cel/BUILD.bazel b/cel/BUILD.bazel
index b43beed..aabc323 100644
--- a/cel/BUILD.bazel
+++ b/cel/BUILD.bazel
@@ -28,7 +28,7 @@ go_library(
         "//interpreter:go_default_library",
         "//interpreter/functions:go_default_library",
         "//parser:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//reflect/protodesc:go_default_library",
         "@org_golang_google_protobuf//reflect/protoreflect:go_default_library",
@@ -64,8 +64,8 @@ go_test(
         "//test:go_default_library",
         "//test/proto2pb:go_default_library",
         "//test/proto3pb:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@io_bazel_rules_go//proto/wkt:descriptor_go_proto",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
         "@org_golang_google_protobuf//types/known/structpb:go_default_library",
     ],
 )
diff --git a/checker/BUILD.bazel b/checker/BUILD.bazel
index bec40b6..1deaba7 100644
--- a/checker/BUILD.bazel
+++ b/checker/BUILD.bazel
@@ -30,7 +30,7 @@ go_library(
         "//common/types/pb:go_default_library",
         "//common/types/ref:go_default_library",
         "//parser:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//types/known/emptypb:go_default_library",
         "@org_golang_google_protobuf//types/known/structpb:go_default_library",
diff --git a/checker/decls/BUILD.bazel b/checker/decls/BUILD.bazel
index 5a24f1d..de067ae 100644
--- a/checker/decls/BUILD.bazel
+++ b/checker/decls/BUILD.bazel
@@ -13,7 +13,7 @@ go_library(
     ],
     importpath = "github.com/google/cel-go/checker/decls",
     deps = [
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//types/known/emptypb:go_default_library",
         "@org_golang_google_protobuf//types/known/structpb:go_default_library",
     ],
diff --git a/common/BUILD.bazel b/common/BUILD.bazel
index a0058ae..3c1b00b 100644
--- a/common/BUILD.bazel
+++ b/common/BUILD.bazel
@@ -17,7 +17,7 @@ go_library(
     importpath = "github.com/google/cel-go/common",
     deps = [
         "//common/runes:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_x_text//width:go_default_library",
     ],
 )
diff --git a/common/containers/BUILD.bazel b/common/containers/BUILD.bazel
index 18142d9..7ab7767 100644
--- a/common/containers/BUILD.bazel
+++ b/common/containers/BUILD.bazel
@@ -12,7 +12,7 @@ go_library(
     ],
     importpath = "github.com/google/cel-go/common/containers",
     deps = [
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
     ],
 )
 
@@ -26,6 +26,6 @@ go_test(
         ":go_default_library",
     ],
     deps = [
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
     ],
 )
diff --git a/common/debug/BUILD.bazel b/common/debug/BUILD.bazel
index cf5c5d2..07e48fe 100644
--- a/common/debug/BUILD.bazel
+++ b/common/debug/BUILD.bazel
@@ -13,6 +13,6 @@ go_library(
     importpath = "github.com/google/cel-go/common/debug",
     deps = [
         "//common:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
     ],
 )
diff --git a/common/types/BUILD.bazel b/common/types/BUILD.bazel
index 1f77c86..1847312 100644
--- a/common/types/BUILD.bazel
+++ b/common/types/BUILD.bazel
@@ -38,8 +38,8 @@ go_library(
         "//common/types/ref:go_default_library",
         "//common/types/traits:go_default_library",
         "@com_github_stoewer_go_strcase//:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
-        "@org_golang_google_genproto//googleapis/rpc/status:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
+        "@go_googleapis//google/rpc:status_go_proto",
         "@org_golang_google_grpc//codes:go_default_library",
         "@org_golang_google_grpc//status:go_default_library",
         "@org_golang_google_protobuf//encoding/protojson:go_default_library",
@@ -79,7 +79,7 @@ go_test(
         "//common/types/ref:go_default_library",
         "//test:go_default_library",
         "//test/proto3pb:test_all_types_go_proto",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//encoding/protojson:go_default_library",
         "@org_golang_google_protobuf//types/known/anypb:go_default_library",
         "@org_golang_google_protobuf//types/known/durationpb:go_default_library",
diff --git a/common/types/pb/BUILD.bazel b/common/types/pb/BUILD.bazel
index f23ac9c..fc142f4 100644
--- a/common/types/pb/BUILD.bazel
+++ b/common/types/pb/BUILD.bazel
@@ -17,7 +17,7 @@ go_library(
     ],
     importpath = "github.com/google/cel-go/common/types/pb",
     deps = [
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//encoding/protowire:go_default_library",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//reflect/protoreflect:go_default_library",
diff --git a/common/types/ref/BUILD.bazel b/common/types/ref/BUILD.bazel
index 1d0f468..73e84b3 100644
--- a/common/types/ref/BUILD.bazel
+++ b/common/types/ref/BUILD.bazel
@@ -13,7 +13,7 @@ go_library(
     ],
     importpath = "github.com/google/cel-go/common/types/ref",
     deps = [
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//reflect/protoreflect:go_default_library",
     ],
diff --git a/ext/BUILD.bazel b/ext/BUILD.bazel
index 582a79a..1d186f8 100644
--- a/ext/BUILD.bazel
+++ b/ext/BUILD.bazel
@@ -20,7 +20,7 @@ go_library(
         "//common/types/ref:go_default_library",
         "//common/types/traits:go_default_library",
         "//interpreter/functions:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
     ],
 )
 
diff --git a/interpreter/BUILD.bazel b/interpreter/BUILD.bazel
index 04a3ec7..5a31467 100644
--- a/interpreter/BUILD.bazel
+++ b/interpreter/BUILD.bazel
@@ -32,7 +32,7 @@ go_library(
         "//common/types/ref:go_default_library",
         "//common/types/traits:go_default_library",
         "//interpreter/functions:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//types/known/durationpb:go_default_library",
         "@org_golang_google_protobuf//types/known/structpb:go_default_library",
@@ -65,7 +65,7 @@ go_test(
         "//test:go_default_library",
         "//test/proto2pb:go_default_library",
         "//test/proto3pb:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//types/known/anypb:go_default_library",
     ],
diff --git a/parser/BUILD.bazel b/parser/BUILD.bazel
index b76e6e4..3ff9ba3 100644
--- a/parser/BUILD.bazel
+++ b/parser/BUILD.bazel
@@ -24,7 +24,7 @@ go_library(
         "//common/runes:go_default_library",
         "//parser/gen:go_default_library",
         "@com_github_antlr_antlr4_runtime_go_antlr//:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//types/known/structpb:go_default_library",
     ],
diff --git a/server/BUILD.bazel b/server/BUILD.bazel
index 4629ca4..fddf409 100644
--- a/server/BUILD.bazel
+++ b/server/BUILD.bazel
@@ -18,9 +18,9 @@ go_library(
         "//common/types/ref:go_default_library",
         "@com_google_cel_spec//proto/test/v1/proto2:test_all_types_go_proto",
         "@com_google_cel_spec//proto/test/v1/proto3:test_all_types_go_proto",
-        "@org_golang_google_genproto//googleapis/api/expr/conformance/v1alpha1:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
-        "@org_golang_google_genproto//googleapis/rpc/status:go_default_library",
+        "@go_googleapis//google/api/expr/conformance/v1alpha1:conformance_go_proto",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
+        "@go_googleapis//google/rpc:status_go_proto",
         "@org_golang_google_grpc//codes:go_default_library",
         "@org_golang_google_grpc//status:go_default_library",
         "@org_golang_google_protobuf//proto:go_default_library",
@@ -41,8 +41,8 @@ go_test(
         "//common/operators:go_default_library",
         "//test:go_default_library",
         "@com_google_cel_spec//tools/celrpc:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/conformance/v1alpha1:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
-        "@org_golang_google_genproto//googleapis/rpc/status:go_default_library",
+        "@go_googleapis//google/api/expr/conformance/v1alpha1:conformance_go_proto",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
+        "@go_googleapis//google/rpc:status_go_proto",
     ],
 )
diff --git a/test/BUILD.bazel b/test/BUILD.bazel
index 057beaa..233dad7 100644
--- a/test/BUILD.bazel
+++ b/test/BUILD.bazel
@@ -21,7 +21,7 @@ go_library(
     importpath = "github.com/google/cel-go/test",
     deps = [
         "//common/operators:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/expr/v1alpha1:go_default_library",
+        "@go_googleapis//google/api/expr/v1alpha1:expr_go_proto",
         "@org_golang_google_protobuf//proto:go_default_library",
         "@org_golang_google_protobuf//types/known/structpb:go_default_library",
     ],
-- 
2.25.1

